app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,11 @@
+class MessagesController < ApplicationController
+  
+  def index
+    @messages = current_company.messages
+    
+  end
+
+  def create
+    @message = Message.create!(params.require(:message).permit([:message, :employee_id, :company_id]))
```

Comment:
create a separate function for 
`params.require(:message).permit([:message, :employee_id, :company_id])`

Reply:
just calling once still be required?